packages/at_secondary_server/lib/src/verb/handler/enroll_verb_handler.dart

```diff
@@ -217,33 +256,34 @@ class EnrollVerbHandler extends AbstractVerbHandler {
 
   bool _doesEnrollmentHaveManageNamespace(
       EnrollDataStoreValue enrollDataStoreValue) {
-    for (var enrollmentNamespace in enrollDataStoreValue.namespaces) {
-      if (enrollmentNamespace.name == enrollManageNamespace) {
-        return true;
-      }
-    }
-    return false;
+    return enrollDataStoreValue.namespaces.containsKey(enrollManageNamespace);
   }
 
-  Future<void> _storeNotification(String notificationKey, String atSign) async {
+  Future<void> _storeNotification(
```

Comment:
Please add method comment explaining what this is doing

Reply:
done.
https://github.com/atsign-foundation/at_server/pull/1496/commits/b22ea2028164aa54299758080d3b9033bf3a98e2